frontend/src/components/Plan/PlanDescription.tsx

```diff
@@ -0,0 +1,32 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// import { useState } from 'react';
+import { icon } from '@fortawesome/fontawesome-svg-core/import.macro'
+import styles from './PlanDescription.module.css'
+
+interface Props {
```

Comment:
generally good practice when typing the props for a `Component` to name it `ComponentProps` in this case it'd be `PlanDescriptionProps`

Reply:
done, will change this for other components too